driver/src/main/java/org/neo4j/driver/Driver.java

```diff
@@ -155,6 +155,14 @@
     @Experimental
     Metrics metrics();
 
+    /**
+     * Returns true if the driver metrics reporting is enabled via {@link Config.ConfigBuilder#withDriverMetrics()}, otherwise false.
+     *
+     * @return true if the metrics reporting is enabled.
+     */
+    @Experimental
```

Comment:
Remove `@Experimental`?

```suggestion
    @Experimental
```

Reply:
The metrics is still experimental as it is only available in java. We want to introduce metrics in all drivers with a unified API. As it is only made into java for now. We mark it experimental so that the signature might be changed if we need to adjust for a unified API in the future.